### DISCUSSION TITLE
Version variables in a file's header

### DISCUSSION THREAD

**Discussion by pj483** [community/community#41006] [New to GitHub]
Hi.

I'm new to using GitHub as a version control system and wondered if there's a facility whereby the version of the file can be automatically updated in a file's header? In SCCS (yes; I'm that old) it was possible to maintain the sccsid variable using %I% which SCCS would replace with the version number when the file was retrieved. Is there anything similar in GitHub's version control to maintain the value in a header?

Thanks.

**Comment by ossanna16** [comment]
Hi there @pj483 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
